packages/react-icons/convert.js

```diff
@@ -135,10 +135,10 @@ function processFolder(srcPath, destPath, resizable) {
       var jsCode = 
 `
 
-const ${destFilename}Icon = (props: FluentIconsProps) => {
+const ${destFilename}Icon = React.forwardRef((props: FluentIconsProps, ref: React.LegacyRef<SVGSVGElement>) => {  
```

Comment:
This should be `React.Ref` instead of `React.LegacyRef`. Also, why is this using `SVGSVGElement` here, but just `SVGElement` in wrapIcon?